packages/marko-web-inquiry/browser/fields/country.vue

```diff
@@ -785,5 +786,10 @@ export default {
       default: false,
     },
   },
+  computed: {
```

Comment:
Don't use computed for this -- if you're going to wrap the translator function, use `methods`:
```
  methods: {
    translate(key) {
      return i18n(this.lang, key);
    },
  },
```